README.md

```diff
@@ -178,7 +178,7 @@ such as [ignoring certain matches](https://github.com/anchore/grype#specifying-m
 
 We love contributions, feedback, and bug reports. For issues with the invocation of this action, file [issues](https://github.com/anchore/scan-action/issues) in this repository.
 
-For contributing, see [Contributing](CONTRIBUTING.rst).
+For contributing, see [Contributing](CONTRIBUTING.md).
```

Comment:
Wow, what a good catch! 🙌

Reply:
It'd be cool if there were a "markdown compiler/checker". I'm sure there is somewhere...

Reply:
yeah, i'll add an issue to research that if we get some time